Service/Imap.php

```diff
@@ -92,6 +105,41 @@ protected function getMailbox(string $name): Mailbox
         );
     }
 
+
+    /**
+     * Get new mailboxes instance.
+     *
```

Comment:
I think this could be added to both functions
@return Mailbox[]

Reply:
added